index.rst

```diff
@@ -7,652 +7,415 @@ Table of contents
+   developer/debugging/test-bench
+   developer/debugging/automated-tests
+   developer/debugging/vm-interface
+   developer/debugging/windows-debugging
+   developer/debugging/safe-remote-ttys
+   developer/debugging/mount-lvm-image
 
 
-
-Building
-^^^^^^^^
-
+.. _developers-building:
 
 .. toctree::
    :maxdepth: 1
+   :caption: Developers - building
 
-   Qubes builder </developer/building/qubes-builder>
-
-   Qubes builder details </developer/building/qubes-builder-details>
-
-   Development workflow </developer/building/development-workflow>
-
-   Qubes ISO building </developer/building/qubes-iso-building>
-
+   developer/building/qubes-builder
+   developer/building/qubes-builder-details
+   developer/building/development-workflow
+   developer/building/qubes-iso-building
    Qubes template configs <https://github.com/QubesOS/qubes-template-configs>
 
 
-
-Releases
-^^^^^^^^
-
+.. _developers-releases:
 
 .. toctree::
    :maxdepth: 1
+   :caption: Developers - releases
 
-   Release notes </developer/releases/notes>
-
-   Release schedules </developer/releases/schedules>
-
-   Release checklist </developer/releases/todo>
-
-   Version scheme </developer/releases/version-scheme>
-
+   developer/releases/notes
+   developer/releases/schedules
+   developer/releases/todo
+   developer/releases/version-scheme
 
 
 External Documentation
 ^^^^^^^^^^^^^^^^^^^^^^
 
-
 Unofficial, third-party documentation from the Qubes community and
 others.
 
-Operating System Guides
-^^^^^^^^^^^^^^^^^^^^^^^
-
+.. _external-operating-system-guides:
 
 .. toctree::
    :maxdepth: 1
-
-   Windows qubes <https://github.com/Qubes-Community/Contents/blob/master/docs/os/windows/windows.md>
+   :caption: External - Operating System Guides
 
    Template: Ubuntu <https://github.com/Qubes-Community/Contents/blob/master/docs/os/ubuntu.md>
-
    Template: Whonix <https://github.com/Qubes-Community/Contents/blob/master/docs/privacy/whonix.md>
-
    Template: CentOS <https://github.com/Qubes-Community/Contents/blob/master/docs/os/centos.md>
-
    Template: Gentoo <https://github.com/Qubes-Community/Contents/blob/master/docs/os/gentoo.md>
-
    Pentesting <https://github.com/Qubes-Community/Contents/blob/master/docs/os/pentesting.md>
-
    Pentesting: BlackArch <https://github.com/Qubes-Community/Contents/blob/master/docs/os/pentesting/blackarch.md>
-
    Pentesting: Kali <https://github.com/Qubes-Community/Contents/blob/master/docs/os/pentesting/kali.md>
-
    Pentesting: PTF <https://github.com/Qubes-Community/Contents/blob/master/docs/os/pentesting/ptf.md>
-
    Tips for Using Linux in an HVM <https://github.com/Qubes-Community/Contents/blob/master/docs/os/linux-hvm-tips.md>
-
    Creating a NetBSD VM <https://github.com/Qubes-Community/Contents/blob/master/docs/os/netbsd.md>
 
 
-
-Security Guides
-^^^^^^^^^^^^^^^
-
+.. _exteranl-security-guides:
 
 .. toctree::
    :maxdepth: 1
+   :caption: External - Security Guides
 
```

Comment:
QA: in the original index the order is as follows here under Sec: 
[Multi-factor Login](https://www.qubes-os.org/doc/mfa/)
[CTAP proxy](https://www.qubes-os.org/doc/ctap-proxy/)